crates/pink/src/contract.rs

```diff
@@ -137,6 +139,7 @@ impl Contract {
             salt,
             block_number,
             now,
+            None,
```

Comment:
No log in this case?

Reply:
This fn is only for unittest. no log is needed.